astropy/tests/tests/test_runner.py

```diff
@@ -1,9 +1,11 @@
-from astropy.tests.runner import TestRunner, TestRunnerBase, keyword
```

Comment:
Note: For some reason, importing TestXXX into namespace here makes pytest 3 discover them as separate tests and then complain about them. Renaming them on import seems to get rid of the warnings on Travis.